### DISCUSSION TITLE
implementation details for propagation modules in  ViTDet

### DISCUSSION THREAD

**Discussion by wenmengzhou** [alibaba/EasyCV#65] [Ideas]
We have implemented ViT-Det with global propagation in #3 ,   but not provided the implementation of convolutional propagation, which involves MAE pretraining.  As we have implemented MAE, it is straghtforward to try to implement this module before the official code is release.  @tuofeilunhifi 

The mainly description is as follows

![image](https://user-images.githubusercontent.com/4771825/168713959-1abd3075-40ce-4c31-88cb-6c2cac15d4d9.png)

**Comment by wenmengzhou** [comment]
@tuofeilunhifi   "Our backbone adaptation is simple and makes detection fine-tuning compatible with global self-attention pre-training. As stated, it is not necessary to
redesign the pre-training architectures."   from this statement,  we may only add conv propagation during the finetuning stage, and leaving MAE preatraining as it is ？

**Reply by Michaell109** [reply]
Yeah, my understanding of the statement is the same as yours. The conv propagation is only needed in finetuning stage. By the way, would you like to share the mAP of ViTDet (base) you implemented? It would be very helpful for me. Thanks very much.

**Reply by wenmengzhou** [reply]
sure, as we finished refactoring some code, we will release both code and models, maybe expected at next week

**Reply by Michaell109** [reply]
Thanks! I am looking forward to it.

**Reply by tuofeilunhifi** [reply]
> Yeah, my understanding of the statement is the same as yours. The conv propagation is only needed in finetuning stage. By the way, would you like to share the mAP of ViTDet (base) you implemented? It would be very helpful for me. Thanks very much.

The accuracy of conv aggreation we reproduced is 42.13, so we guess that pre-training of conv aggregation is required
